dlc-test/src/test/scala/org/bitcoins/dlc/DLCTestVectorTest.scala

```diff
@@ -0,0 +1,44 @@
+package org.bitcoins.dlc
+
+import org.bitcoins.dlc.testgen.{
+  DLCTestVector,
+  DLCTestVectorGen,
+  DLCTxGen,
+  SuccessTestVector
+}
+import org.bitcoins.testkit.util.BitcoinSAsyncTest
+import org.scalacheck.Gen
+
+import scala.concurrent.Future
+
+class DLCTestVectorTest extends BitcoinSAsyncTest {
+  behavior of "DLCTestVectors"
+
+  val randomTestVectorGen: Gen[Future[DLCTestVector]] =
```

Comment:
This could be moved to `DLCTestVectorGen`

Reply:
`DLCTestVectorGen` is not in testkit or test scope so that's not the best place for it. Eventually I do plan on moving it to testkit though, but not until I have more types of `DLCTestVector` and have hashed out how I'm going to distinguish them and such